src/plugins/janus_audiobridge.c

```diff
@@ -6326,32 +6328,32 @@ void janus_audiobridge_incoming_rtp(janus_plugin_session *handle, janus_plugin_r
 	if(!session || g_atomic_int_get(&session->destroyed) || !session->participant)
 		return;
 	janus_audiobridge_participant *participant = (janus_audiobridge_participant *)session->participant;
-	if(!g_atomic_int_get(&participant->active) || participant->muted || g_atomic_int_get(&participant->suspended) ||
-			(participant->codec == JANUS_AUDIOCODEC_OPUS && !participant->decoder) || !participant->room)
+	if(!g_atomic_int_get(&participant->active) || participant->muted || g_atomic_int_get(&participant->suspended) || !participant->room)
```

Comment:
Why did you remove the check on the decoder there?

Reply:
At first, I added the following code above it:
```
gboolean opus_decoding = FALSE;
janus_mutex_lock(&participant->decoding_mutex);
if(participant->decoder)
	opus_decoding = TRUE;
janus_mutex_unlock(&participant->decoding_mutex);
```
And then used `opus_decoding` instead of  `participant->decoder` in the checks. But this would require an extra lock each time.

I also though of just keeping it like it was, but that could cause some memory fence issue (thread uses stale value) I think?

Since the packet would be dropped later on if `participant->decoder` is NULL, I removed the extra check.